core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/core/query/Query.java

```diff
@@ -246,7 +256,7 @@ public String getQueryString() {
 		if (offset >= 0) {
 			query.append(OFFSET + " ").append(offset).append("\n");
 		}
-
+		SparqlBuilderUtils.appendAndNewlineIfPresent(values, query);
```

Comment:
Doesn't the VALUES clause usually go at the beginning of the WHERE clause?

Reply:
No.  https://www.w3.org/TR/sparql11-query/#sparqlGrammar says:

``` 
[2] | Query  ::=  Prologue
                  ( SelectQuery | ConstructQuery | DescribeQuery | AskQuery )
                  ValuesClause
```